scripts/irods/test/test_iadmin.py

```diff
@@ -1603,6 +1603,66 @@ def test_modresc_host_path_warning_messages(self):
         # Changing vault should have failed, so make sure it has not changed
         self.admin.assert_icommand(['ilsresc', '-l', self.resc_name], 'STDOUT_SINGLELINE', 'vault: ' + self.good_vault)
 
+    def generate_invalid_resource_names(self, prefix):
+        return [
+            '{0} Resc'.format(prefix),
+            '{0}\fResc'.format(prefix),
+            '{0}\nResc'.format(prefix),
+            '{0}\rResc'.format(prefix),
+            '{0}\tResc'.format(prefix),
+            '{0}\vResc'.format(prefix)
+        ]
+
+    def test_creating_a_resource_with_whitespace_is_not_allowed__issue_5861(self):
```

Comment:
didn't we make a `setparentresc` also?

Reply:
That is part of the atomic database operations PR. It is still planned, but the atomic PR has taken a backseat to other things.

Reply:
ah, yes, of course.